openwisp_radius/tests/test_tasks.py

```diff
@@ -173,6 +172,25 @@ def test_send_login_email(self, translation_activate, logger):
                 ' and/or traffic has been used or you can terminate the session',
                 ' '.join(email.alternatives[0][0].split()),
             )
+            self.assertNotIn(
+                'NOTE: This link is valid only for an hour from now',
+                ' '.join(email.alternatives[0][0].split()),
+            )
+            translation_activate.assert_called_with(user.language)
+
+        translation_activate.reset_mock()
+
+        with self.subTest(
+            'it should check expiration text is present when SESAME_MAX_AGE is set'
+        ):
+            setattr(settings, 'SESAME_MAX_AGE', 2 * 60 * 60)
```

Comment:
> I would mock this settings as following instead of using `setattr`.
> 
> ```
> mock.patch('sesame.settings.MAX_AGE')
> ```
> 
> If it is not possible to mock this setting, then let's unset this after completion of the test. Otherwise, it could affect other tests as well.

Can we also do something like this? 🤔

```python
from django.test.utils import override_settings

    with override_settings(SESAME_MAX_AGE=2*60*60):
        with self.subTest(
                'it should check expiration text is present when SESAME_MAX_AGE is set'
            ):
            # Test code
```
https://docs.djangoproject.com/en/4.0/topics/testing/tools/#overriding-settings